security.rst

```diff
@@ -2364,14 +2365,21 @@ anonymous users access by checking if there is no user set on the token::
         {
             // ...
 
-            if (!$token->getUser() instanceof UserInterface) {
+            if ($token instanceof NullToken) {
                 // the user is not authenticated, e.g. only allow them to
                 // see public posts
                 return $subject->isPublic();
             }
         }
     }
 
+.. caution::
+
+    :class:`Symfony\\Component\\Security\\Core\\Authentication\\Token\\NullToken` is only available in voters
+    (because :method:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface::vote` can't          receive a null token). Outside of voters (controllers, other services...) there is no token in the
```

Comment:
```suggestion
    (because the :method:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface::vote` method can't receive a null token). Outside of voters (controllers, other services...) there is no token in the
```